services/web/client/source/resource/qxapp/node-meta-v1.0.0.json

```diff
@@ -0,0 +1,331 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://osparc.io/schema/node-meta-v1.0.0",
+    "title": "simcore node",
+    "description": "Description of a simcore node 'class' with output definitions",
+    "type": "object",
+    "additionalProperties": false,
+    "required": [
+        "key",
+        "version",
+        "type",
+        "name",
+        "description",
+        "authors",
+        "contact",
+        "$schema"
+    ],
+    "properties": {
+        "$schema": {
+          "type": "string",
```

Comment:
when it is present, everyone knows which schema this node description is using AND it is sort of a standard to have $schema in the json structure as it will help editing tools to pick up the schema of a random json file and thus improve editing ... VC is such a tool